typescript_fibonacci.ts

```diff
@@ -0,0 +1,12 @@
+const getFibonacciNumbers = (seriesLength: number): number[] => {
+  //Will always start with 0, 1, so we can safely initialize.
+  const seriesArr: number[] = [0, 1]
+
+  //We added the two initial numbers, so remove 2 from series length to compensate.
+  for (let i = 1; i <= seriesLength - 2; i++) {
+    seriesArr.push(seriesArr[i - 1] + seriesArr[i])
```

Comment:
Codacy found an issue: [Generic Object Injection Sink](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6288784)

Reply:
Generic Object Injection does not apply here, because 'i' is not accessible to the user.